docs/operation.md

```diff
@@ -35,7 +35,7 @@ If the node is unresponsive, inspect the `log` directory for errors.
 
 Back up the peer key pair:
 ```bash
-cp -pr ~/aeternity/node/keys ~/my_aeternity_keys
+cp -pr ~/aeternity/node/data/aecore/keys ~/my_aeternity_keys
```

Comment:
I see this corresponds to [the code](https://github.com/aeternity/aeternity/blob/master/apps/aecore/src/aec_keys.erl#L516)